framework/commercejs/customer/use-customer.tsx

```diff
@@ -0,0 +1,44 @@
+import Cookies from 'js-cookie'
+import { decode } from 'jsonwebtoken'
+import { SWRHook } from '@commerce/utils/types'
+import useCustomer, { UseCustomer } from '@commerce/customer/use-customer'
+import { CUSTOMER_COOKIE, API_URL } from '@framework/constants'
```

Comment:
We shouldn't import from framework inside the provider as it might break other providers